xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -168,14 +173,11 @@ public void handleSubchannelState(Subchannel subchannel, ConnectivityStateInfo n
     if (fallbackManager.fallbackBalancer != null) {
       fallbackManager.fallbackBalancer.handleSubchannelState(subchannel, newState);
     }
-    if (subchannelStore.hasSubchannel(subchannel)) {
-      if (newState.getState() == IDLE) {
-        subchannel.requestConnection();
-      }
-      subchannel.getAttributes().get(STATE_INFO).set(newState);
-      xdsLbState.handleSubchannelState(subchannel, newState);
-      fallbackManager.maybeUseFallbackPolicy();
+    if (newState.getState() == IDLE) {
+      subchannel.requestConnection();
```

Comment:
Child policies already take care of it. It not necessary on the xDS balancer.

Reply:
Done.